README.md

```diff
@@ -611,6 +611,41 @@ $ niet project.meta.name tests/samples/sample.yaml
 my-project
 ```
 
+### Complexe search
+
+Consider the following content:
+
+```
+$ cat /var/lib//libvirt/dnsmasq/virbr0.status
+[
+  {
+    "ip-address": "192.168.122.113",
+    "mac-address": "52:54:00:91:14:02",
+    "hostname": "rhel79",
+    "expiry-time": 1644251254
+  },
+  {
+    "ip-address": "192.168.122.162",
+    "mac-address": "52:54:00:23:37:ed",
+    "hostname": "satellite",
+    "expiry-time": 1644251837
+  }
+]
+```
+
+Here we want to retrieve the value of the ip-address field when the hostname
+is equal to `satellite`. The following command will allow you to get this
+value:
+
+```
+$ sed 's/ip-address/ip/g' /var/lib//libvirt/dnsmasq/virbr0.status | niet "[?hostname=='satellite'].ip"
```

Comment:
We should replace  sed 's/ip-address/ip/g' by  sed 's/ip/_/g'
It is more consistent to niet without any '-'